elastica/boundary_conditions.py

```diff
@@ -427,9 +429,7 @@ def nb_constrain_rotational_rates(omega_collection, indices) -> None:
         block_size = indices.size
         for i in range(block_size):
             k = indices[i]
-            omega_collection[0, k] = 0.0
-            omega_collection[1, k] = 0.0
-            omega_collection[2, k] = 0.0
+            omega_collection[..., k] = 0.0
```

Comment:
ditto ^

Reply:
Fixed in https://github.com/GazzolaLab/PyElastica/pull/147/commits/9a95c4389de20fce48cc9771faeb714f4694607c